ge_worker/bonuses/bonus_tasks.py

```diff
@@ -0,0 +1,8 @@
+from bonuses.utils import celery_app
+from bonuses.bonus_processor import BonusProcessor
+
+
+@celery_app.task()
+def auto_disburse_daily_bonus():
+    bonus_processor = BonusProcessor(issuable_amount=1000)
```

Comment:
Should this be hardcoded to `1000`?

Reply:
The idea is to have it configurable but we need to agree on whether we're setting a GUI for the rewards system before that can be baked into this. It should come as part of this [Write spec for rewards system #13](https://github.com/GrassrootsEconomics/CIC-Docs/issues/13)
Consider it a test value until we're agreed on how to proceed on that.
Essentially the final `Bonus processor` should allow for configurable time frames as well as minimum amounts.